canonical-web-and-design/ubuntu.com
# Display the contract token

## Done

- Added hooks etc. for fetching the contract token.
- Fetch and display the token in the details panel.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10330.demos.haus/advantage?test_backend=true) and log in.
- Click on a subscription the left.
- The panel on the right should display the token in a code block under the "Subscription" title.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/223.

## Screenshots

<img width="684" alt="Screen Shot 2021-09-09 at 1 44 16 pm" src="https://user-images.githubusercontent.com/361637/132619007-ef32c568-1b66-4b6e-b388-a7b842978580.png">
